oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/Rules.java

```diff
@@ -31,13 +31,15 @@
 @Getter
 @ToString
 public class Rules {
+    private List<CompositeAlarmRule> compositeRules;
```

Comment:
This should move to the last field of the class.

Reply:
done